### ISSUE TITLE
Viewing entry with multiple auto-type window associations evokes unsaved changes.

### ISSUE THREAD

**Issue by TheCayman** [keepassxreboot/keepassxc#4182]
**Expected Behavior**

Should be able to fully view windows associations of an entry without making any changes and not be prompted about "unsaved changes" when leaving the entry.

**Current Behavior**
If you view the window associations by clicking on them, you can't leave that entry without getting the warning about unsaved changes (with nothing being changed).

**Steps to Reproduce**
1. Open an entry with multiple Auto-Type Window Associations.
2. Click Auto-Type
3. The first Association will be highlighted in grey. Select the second one.
4. Press Cancel

**Context**
Just normal use. Work with a new database with just a single entry.

**Debug Info**
Version: 2.5.2 64-bit on Windows 10.10.19536

Remarks:
[#3013](https://github.com/keepassxreboot/keepassxc/issues/3013) is quite similar, except that the fix for that one obviously did not solve this one.

For some reason, I did not get the bug-reporting template when creating this issue. I have therefore tried to mimic the template.

**Comment by droidmonkey** [comment]
Good catch

**Comment by TheCayman** [comment]
If you press save when being presented the "Entry has unsaved changes" pop-up, the status of the database does **_not_** change to changed (ie the save button remains greyed out and no star appears after the name of the database).

**Comment by droidmonkey** [comment]
That is because nothing actually changed. This is just a gui signaling issue where we regard changing the selection of an element to be a dirty change event.

**Comment by droidmonkey** [comment]
This was fixed in 2.6.0

**Comment by TheCayman** [comment]
I just tested this on 2.6.0 and it appear not to have been fixed. Do I need to create another issue or can we continue using this one?

KeePassXC - Version 2.6.0
Revision: 0765954

Qt 5.15.0
Debugging mode is disabled.

Operating system: Windows 10 Version 1909
CPU architecture: x86_64
Kernel: winnt 10.0.18363

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey

Cryptographic libraries:
 libgcrypt 1.8.5
